### PR TITLE
chore(config): ng-config

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

File: addon/ng2/models/config.ts [hunk 1]
```diff
@@ -27,9 +24,22 @@ function _findUp(name: string, from: string) {
 
 
 export class CliConfig {
-  constructor(private _config: CliConfigJson = CliConfig.fromProject()) {}
+  _config: any;
```
Comment (hansl) [inline addon/ng2/models/config.ts]: `private`?

Comment (jkuri) [inline addon/ng2/models/config.ts]: Yes, we set variables with `set()` method and not with direct allocation as we don't use proxies now.

Comment (hansl) [inline addon/ng2/models/config.ts]: Could you make that private?

Comment (jkuri) [inline addon/ng2/models/config.ts]: Sorry, forgot the keyword. Done.

**Comment by hansl** [comment]
LGTM. Let's get that in.

**Pull request merged**